src/implementations/vanilla/src/basics/flyout/tests/test-flyout_ja.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang='ja' >
+    <head>
+        <meta charset="utf-8"/>
+        <link rel="stylesheet" href="../../../../lib/hig.css">
+        <script type="text/javascript" src="../../../../lib/hig.js"></script>
+        <style>
+            .test-container {
+                padding: 40px;
+                margin: 0;
+                display: flex;
+                justify-content: center;
+            }
+        </style>
+    </head>
+    <body>
+        <div class='test-container'></div>
+
+        <script>
+            var i18next = Hig.i18next;
+            i18next.changeLanguage('ja');
+
+            function createFlyoutContent() {
+                var flyoutContent = document.createElement('div');
+                flyoutContent.textContent = i18next.t("general.hello_world");
+                return flyoutContent;
+            }
+
+
+            // ----------------------------
+            // DEFAULT FLYOUT
+            // ----------------------------
+
+            var defaultFlyout = new Hig.Flyout({
+                anchorPoint: 'top-right'
+            });
+            defaultFlyout.mount('.test-container');
+            defaultFlyout.addSlot(createFlyoutContent());
+            defaultFlyout.onClickOutside(function() {
+                defaultFlyout.close();
+            });
+
+            var defaultButton = new Hig.Button({
+                "title": i18next.t("flyout.open_setter_btn")
```

Comment:
Same as above, should be default button key.

Reply:
Yes, good catch.